ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -22,6 +22,10 @@
 from google.protobuf.internal.encoder import _EncodeVarint  # type: ignore
 
 
+INITIAL_POS = 33
```

Comment:
Does this mean we should fix INITIAL_POS for all future versions?

Reply:
Depends - I think the current meta proto should stay as it is. But if we wanted to add more data in a v2 format, we could have a v2 metadata that's in file after the v1 metadata.